### ISSUE TITLE
for the rootview , NewHandler Draw func does not work

### ISSUE THREAD

**Issue by liangminghaoAngus** [yohamta/furex#61]
i try to write the code as example used
```
view := furex.Parse(onBoardHtml, &furex.ParseOptions{...
Handler: furex.NewHandler(furex.HandlerOpts{
		HandleRelease: func(x, y int, isCancel bool) {},
		HandlePress:   func(x, y int, t ebiten.TouchID) {},
		Update:        func(v *furex.View) {},
		Draw: func(screen *ebiten.Image, frame image.Rectangle, v *furex.View) {
			// screen.Fill(Hex2RGB("#0dceda", 0))
			sw, sh := frame.Min.X+frame.Dx()/2, frame.Min.Y+frame.Dy()/2
			opt := &ebiten.DrawImageOptions{}
			opt.GeoM.Scale(0.6, 0.6)
			scale := ebiten.DeviceScaleFactor()
			opt.GeoM.Scale(scale, scale)
			opt.GeoM.Translate(float64(sw-(bgImgFile.Bounds().Dx()/2)), float64(sh-(bgImgFile.Bounds().Dy()/2)-10))
			opt.Filter = ebiten.FilterLinear

			screen.DrawImage(bgImg, opt)
		},
	}),
})
```

then i debug  `Draw` , found `Draw` does not call , is there any problem for the way i using?

**Comment by yohamta** [comment]
Hello @liangminghaoAngus,

Thanks for sharing the issue. At a glance, I can't see any problem in your code.
One possible cause would be not calling `view.Draw()` but I think it's not likely.

I appreciate if you can could share a minimum reproducible project for further investigation.

**Comment by liangminghaoAngus** [comment]
@yohamta I'm glad to receive your reply，there is a minimum tony project ,using the same `html` file parse
i do the `println()` func in `Draw` and `Update`. 
run `go run .` ,u can see just the `Update println()` call.
[https://github.com/liangminghaoAngus/tony](url)

**Comment by yohamta** [comment]
Hi @liangminghaoAngus,

Thank you so much for providing the reproducible code, it was incredibly helpful.
The issue is fixed in `v2.4.3`. Please give it a try.
